### ISSUE TITLE
[BUG] Stashed files not handled in some cases

### ISSUE THREAD

**Issue by iusmac** [arl/gitmux#66]
Hello again! It appears that stashed files are no longer handled in _v0.7.8_.

**How to reproduce**
```bash
$ mkdir testGitmux
$ cd testGitmux
$ git init
$ touch test_file
$ git add test_file
$ git commit -m 'First commit'
$ touch new_file
$ git add new_file 
$ git stash push
```
Even though there's a stashed file, gitmux claims the tree is "clean". But if we create an untracked file (`touch another_file`), gitmux will show 1 untracked and 1 stashed.

I've checked the previous version (_v0.7.7_) and stashed files are handled as expected.

**Comment by arl** [comment]
Hi.
In latest version, I've slightly changed the definition of a _clean_ repository. It has always bothered me that the clean flag depended on the number of stashed entries. Stash entries could have been added from other branches, and should not cause in my opinion the repository to be considered_dirty_.

However, stash entries should still be counted and displayed in the status, is it still the case for you?

**Comment by iusmac** [comment]
@arl I've been using gitmux for more than a year, and I've really got used to the stashed indicator, for me it's like "list of pending jobs", it helps a lot when switching between projects during the day. I agree that the clean tree does not depend on what's in stash, so I think we should always display stashed files independently, exactly like we do with the remote and the current branch which, indeed, does not affect how "clean" the tree is.

**Comment by arl** [comment]
Thank you again for noticing and reporting!

**Comment by arl** [comment]
Fixed in v0.7.9 release (which is currently building)
